### ISSUE TITLE
PhantomJS with webdriverio

### ISSUE THREAD

**Issue by michaelBenin** [webdriverio/webdriverio#342]
Yosemite 10.10
PhantomJS version: 1.9.8

Configuration:

```
webdriver: {
headless: {
  options: {
    bail: false,
    desiredCapabilities: {
      browserName: 'phantomjs'
    }
  },
  tests: [
    'acceptance_test/init.js'
  ]
  }
```

Uncaught RuntimeError
     (UnknownError:13) An unknown server-side error occurred while processing the command.
     Problem: org.openqa.selenium.browserlaunchers.Proxies

**Comment by christopherbalz** [comment]
Still not working with phantomjs as the browser, even with the latest of selenium-standalone (4.2.0), phantomjs (1.9.16), webdriverio (2.4.5), and jasmine-node (1.14.5).  

Error is:
```
### Execution Started ### Running Test which does foo when bar is clicked.
{ [RuntimeError: RuntimeError
     (UnknownError:13) An unknown server-side error occurred while processing the command.
     Problem: org/openqa/selenium/browserlaunchers/Proxies

     Callstack:
     -> init()
```

**Comment by christian-bromann** [comment]
@christopherbalz This is not a Selenium related problem but more a phantomjs related one. Could you try to update PhantomJS to v2.0 and see if that works?

**Comment by christopherbalz** [comment]
I think this 1.9.16 is the latest release: https://github.com/Medium/phantomjs/releases

**Comment by christian-bromann** [comment]
Phantomjs recently released [v2.0.0](http://phantomjs.org/download.html)

**Comment by christopherbalz** [comment]
Thanks; good idea - tried but no luck; same error as before (had to use the [`upx` package](https://stackoverflow.com/questions/28267809/phantomjs-getting-killed-9-for-anything-im-trying/28890209#28890209) 

```
$ phantomjs --version
2.0.0
```
